### PR TITLE
create task order backfill script

### PR DESCRIPTION
**Pull request by johnclary**
## Associated issues
- https://github.com/cityofaustin/atd-data-tech/issues/14246

## Testing
**URL to test:**  Local

1. Start local with production data
2. Login to local editor with JD's login and copy your `Authorization` token from the graphql request headers.
3. Save `secrets_template.py` as `secrets.py` and add your auth token to the `local` entry.
4. Fix your user ID in the prod database: 

 ```sql
update moped_users set user_id = 1 where first_name = 'JD';
```

5. from a python environment with `requests` installed, run the script

```shell
$ python backfill_task_orders.py -e local
```
6. Success!

---
#### Ship list
- [ ] Code reviewed 
- [ ] Product manager approved
- [ ] Added to [QA test script](https://docs.google.com/spreadsheets/d/1n_O6MLh9cwwPf57HUM394Ea-z9uuoEV1-QW4axNZXLE/edit#) if applicable

### CODE REVIEW

**Comment by johnclary** [comment]
👋 not urgent—I'm still waiting on the go-ahead from AMD

**Review by mddilley — APPROVED** [review]
I see project task orders inserted into project work activity records in my local DB with prod data. Good to go when you get the green light. ✅

**Pull request merged**
